src/App.js

```diff
@@ -27,45 +27,107 @@ class App extends Component {
     this.state = {
       items: [],
       points: 0,
+      onPlay: false,
+      hasBeenPlayed: false,
     };
 
     // Uncomment this to spawn a single test item
-    //const testItem = this.spawnItem(Date.now());
-    //this.state.items.push(testItem);
+    // const testItem = this.spawnItem(Date.now());
+    // this.state.items.push(testItem);
+  }
 
+  startGame = () => {
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
+    this.setState({
+      onPlay: true,
+      hasBeenPlayed: false,
+      points: 0, // so that after the player has played the game once and click
+                //"Start Game" again, the points will be reset to 0
+    });
+  }
 
-    console.log(this.state);
+  stopGame = () => {
+    // Uncomment this to automatically spawn new items
+    this.disableSpawner();
+    this.setState({
+      onPlay: false,
+      hasBeenPlayed: true,
+    });
   }
 
-  onItemClicked = () => {
-    // Fill this in!
+  markLitterSpotted = (itemIndex) => {
+    let updatedItems = this.state.items;
+    if (updatedItems[itemIndex].type === "litter") {
+      updatedItems[itemIndex].isSpotted = true;
+      this.increasePoints();
+    } else {
+      updatedItems[itemIndex].isSpotted = false;
```

Comment:
Hm, it seems like `isSpotted` does not describe if it's been spotted or not (because if it gets here, it HAS been spotted), but it describes if this spotted item is litter... Could we call this something else besides `isSpotted`?